packages/zowe-explorer-api/src/vscode/ui/utils/HTMLTemplate.ts

```diff
@@ -21,10 +21,13 @@ const HTMLTemplate: string = `
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta 
             http-equiv="Content-Security-Policy" 
-            content="default-src 'none'; img-src vscode-resource: https:; script-src 'nonce-{{ nonce }}';
+            content="default-src 'none'; font-src data:; img-src data: vscode-resource: https:; script-src 'nonce-{{ nonce }}';
             style-src vscode-resource: 'unsafe-inline' http: https: data:;"
         />
         <base href="{{ uris.resource.build }}">
+        {{#uris.resource.css}}
+        <link type="text/css" rel="stylesheet" href="{{ uris.resource.css }}" />
+        {{/uris.resource.css}}
```

Comment:
Do we want to add support for CSS resources to webviews in ZE V2, or is this a change that we should postpone until V3?

Reply:
Sorry about that, yeah this should be in the v3 branch only. I've removed it in 09da4a1 👍